docs/overhetalgoritmekader/index.md

```diff
@@ -1,83 +1,34 @@
-Hierna zal het Algoritmekader in 'beheer' worden genomen. Dat betekent dat doorlopend de laatste ontwikkelingen, in afstemming met de omgeving, worden toegevoegd zodat het Algoritmekader actueel en betrouwbaar blijft.  
-
-## Samenwerking met de omgeving
-Een belangrijk uitgangspunt van het Algoritmekader is dat het kader op een open en transparante manier wordt ontwikkeld. 
-Hierbij wordt gebruik gemaakt van verschillende communicatiekanalen, zoals een communityplatform [Pleio](https://algoritmes.pleio.nl/), GitHub, periodieke bijeenkomsten en (waar passend) werkgroepen. 
-Informatie over algoritmes en AI worden uitgewerkt. 
-Zodra deze informatie op 'niveau' is, wordt dit in Github geplaatst en daarmee 'gepubliceerd'. 
-Dit noemen we *releases*. 
-
-De motivatie om op deze ‘open source’ manier aan dit Algoritmekader te werken is enerzijds de omgeving van deze informatie te voorzien en anderzijds om de omgeving uit te nodigen om hier feedback op te geven. 
-Werken in Github is voor het team Algoritmekader nieuw en experimenteel. 
-Het biedt de mogelijkheid voor alle geïnteresseerden om hun kennis en zienswijze te delen. 
-Daarmee kan (nieuwe) kennis snel onderdeel worden gemaakt van het Algoritmekader. 
-Daardoor blijft het Algoritmekader zo actueel mogelijk en wordt de inhoud breed gedragen.
-Tegelijkertijd vraagt dit om een aangepaste werkwijze en is hier bepaalde expertise voor nodig. 
-Het begin is gemaakt en het team Algoritmekader is nog lerende om hier optimaal invulling aan te geven. 
-Voor nu betekent dit concreet dat het langer kan duren voordat wordt gereageerd op suggesties of toevoegingen. 
-Er wordt gewerkt aan het inrichten van goede processen om snel en kundig te reageren. 
+# Over het algoritmekader 
+Het algoritmekader is een hulpmiddel voor overheden die algoritmes of AI (artificiële intelligentie) gebruiken. Hierin zoek je de regels en adviezen op per situatie. De informatie in het Algoritmekader is nog niet betrouwbaar. Het is een testversie. De definitieve versie verschijnt eind 2024.
+
+## Verantwoord gebruik van algoritmes en AI
+In het Algoritmekader vind je informatie over verantwoord gebruik van algoritmes en AI. Bijvoorbeeld:
+- Wie is waarvoor verantwoordelijk?
+- Hoe voorkom je discriminatie (bias) in een algoritme?
+- Hoe verwerk je gegevens op een veilige manier?
+- Welke algoritmes controleer je op mensenrechtenschendingen?
+- Welke inkoopvoorwaarden spreek je af voor algoritmes van een externe partij?
+
+De informatie is bedoeld voor medewerkers van de rijksoverheid, provincies, gemeentes en waterschappen.
+
+## Informatie van de overheid
+Het Algoritmekader is een website van het Ministerie van Binnenlandse Zaken en Koninkrijksrelaties. Het team Data, AI en Algoritmes stelt de informatie samen, op basis van:
+- alle wetten en regels, zoals de [AI-verordening](https://www.digitaleoverheid.nl/overzicht-van-alle-onderwerpen/nieuwe-technologieen-data-en-ethiek/artificiele-intelligentie-ai/ai-verordening/)
```

Comment:
Misschien beter ter verwijzen naar de AI verordening zelf? https://eur-lex.europa.eu/legal-content/NL/TXT/HTML/?uri=OJ:L_202401689#d1e39-1-1

Reply:
Kan ook, maar dan moeten we deze link gebruiken: https://eur-lex.europa.eu/legal-content/NL/TXT/?uri=CELEX%3A32024R1689&qid=1723226664277